.github/workflows/build.yml

```diff
@@ -259,13 +259,14 @@ jobs:
           export BRANCH=$(echo $GITHUB_REF | cut -d'/' -f 3)
           ./appveyor/docker_build/build_docker_image.sh
       - if: ${{ github.event_name == 'push' }}
-        name: Log in to Docker Hub
+        name: Login with Github Container registry
         uses: docker/login-action@v2
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
       - if: ${{ github.event_name == 'push' }}
-        name: Push to ${{ env.DOCKER_REPOSITORY }}
+        name: Push to GitHub Container registry
```

Comment:
Ditto.

Reply:
As said above, I think `GitHub container registry` is a valid term (this is what `ghcr.io` stands for) and what's used in the documentation.